src/Controller/EthereumController.php

```diff
@@ -92,9 +105,11 @@ public static function getServerOptionsArray($filter_enabled = FALSE){
    * Returns Ethereum Servers.
    *
    * @param bool $filter_enabled
-   *   (optional) Restrict to enabled servers. Defaults to FALSE.
+   *    Restrict to enabled servers. Defaults to FALSE (optional).
+   *
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
    *
-   * @return \Drupal\ethereum\EthereumServerInterface[]
+   * @return array of \Drupal\ethereum\Entity\EthereumServer
```

Comment:
What's the reason for this change? The previous code was using the standard notation for an array of objects.

Reply:
![dou](https://giphy.com/embed/8WdsK61D9YOOc)